iothub/device/src/Authentication/ClientAuthenticationWithSharedAccessPolicy.cs

```diff
@@ -49,10 +65,13 @@ public string Key
         /// <summary>
         /// Name of the shared access policy.
         /// </summary>
-        public string PolicyName
+        /// <remarks>
+        /// The name of relevant field in a connection string is "SharedAccessKeyName", so we call the property "KeyName" here.
```

Comment:
This explanation is useful for us internally. However, for the customer this naming justification isn't really required. I think it'll be useful if we point to a connection string format MS doc instead. That'll help users understand which value to pass in here.